aodndata/aims/anmn_nrs.py

```diff
@@ -56,7 +56,7 @@ def get_anmn_nrs_site_name(filepath):
 
     @staticmethod
     def remove_md5_from_filename(filepath):
-        return re.sub('.[0-9a-z]*.nc$', '.nc', filepath)
+        return re.sub('\.[0-9a-z]*.nc$', '.nc', filepath)
```

Comment:
Don't you want to escape the dot in ".nc" too? (i.e. "\.nc") Otherwise it means any character.
Also, since MD5 sums are always 32 characters in lenght, you could fix that in your regex too, i.e.
```regex
'\.[0-9a-z]{32}\.nc$'
```